CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 * Non case-sensitive sorting of title column in Open Loans Screen. Refs UIU-2983.
 * Upload Local File for user profile picture. Refs UIU-2974.
 * Don't allow to display or upload a profile picture to a shadow account. Refs UIU-3045.
+* Implement client-side compression for Profile Pictures. Refs UIU-3064.
```

Comment:
Could you reword it to something like - "Compress selected local file before uploading to server as profile picture"?